src/js/tour.js

```diff
@@ -258,17 +262,12 @@ export class Tour extends Evented {
   }
 
   /**
-   * If we have a currentStep, the tour is active, so just hide the step and remain active.
-   * Otherwise, make the tour active.
+   * Make this tour "active"
    * @private
    */
   _setupActiveTour() {
-    if (this.currentStep) {
-      this.currentStep.hide();
```

Comment:
I'm a little unclear on why we had this hide in the first place, but can you please elaborate on why it is no longer needed?

Reply:
Exactly -- it doesn't seem like it would be a part of `_setupActiveTour`. 

`_setupActiveTour`, however, is called as a part of `next.` So I just moved the `hide` call into `next` to make our intents more clear.